backend/engine/tests/test_trivy.py

```diff
@@ -20,73 +22,6 @@
 
 TEST_IMAGE = os.path.join(TEST_DATA, "image", "test_file_for_docker")
 
-TEST_CHECK_OUTPUT_GEMLOCK_FILE = {
```

Comment:
I assume these tests were removed because the SCA plugin is testing for these lockfiles now, not the image plugin?

Reply:
yes exactly, these tests have been moved to the test_trivy_sca.py file instead as well as some new tests